src/libertem/io/dataset/seq.py

```diff
@@ -269,32 +441,53 @@ def _maybe_load_mrc(self, path):
         data_dict = mrcReader(path)
         return np.squeeze(data_dict['data'])
 
+    def _load_xml_from_file(self, path):
+        if not os.path.exists(path + ".Config.Metadata.xml"):
+            return None
+        if not os.path.exists(path + ".metadata"):
+            return None
+        else:
+            tree = ET.parse(path + ".Config.Metadata.xml")
+            root = tree.getroot()
+            file = open(path + ".metadata", mode="rb")
+            met = file.read()
+            metdata_keys = ['DEMetadataSize', 'DEMetadataVersion', 'UnbinnedFrameSizeX', 'UnbinnedFrameSizeY',
+                            'OffsetX', 'OffsetY', 'HardwareBinning', 'Bitmode', 'FrameRate', 'RotationMode',
+                            'FlipMode', 'OkraMode']
+            metadata = dict(zip(metdata_keys, struct.unpack_from('iiiiiiiiiii?', met, 282)))
+            return xml_processing(root, metadata)
+
     def _maybe_load_dark_gain(self):
         self._dark = self._maybe_load_mrc(self._path + ".dark.mrc")
         self._gain = self._maybe_load_mrc(self._path + ".gain.mrc")
+        self._excluded_pixels = self._load_xml_from_file(path=self._path)
 
     def get_correction_data(self):
         return CorrectionSet(
             dark=self._dark,
             gain=self._gain,
+            excluded_pixels=self._excluded_pixels,
         )
 
+    def get_sig_shape(self):
+        return self._sig_shape
```

Comment:
Is this used anywhere? I think this can be removed (also, `ds.shape.sig` should be easy enough)